modules/luci-mod-status/htdocs/luci-static/resources/view/status/connections.js

```diff
@@ -130,12 +131,27 @@ return view.extend({
 			var src = dns_cache[c.src] || (c.layer3 == 'ipv6' ? '[' + c.src + ']' : c.src);
 			var dst = dns_cache[c.dst] || (c.layer3 == 'ipv6' ? '[' + c.dst + ']' : c.dst);
 
+			let network = c.layer3.toUpperCase();
+			let protocol = c.layer4.toUpperCase();
+			let source ='%h'.format(c.hasOwnProperty('sport') ? (src + ':' + c.sport) : src);
+			let destination = '%h'.format(c.hasOwnProperty('dport') ? (dst + ':' + c.dport) : dst);
+			let transfer = [ c.bytes, '%1024.2mB (%d %s)'.format(c.bytes, c.packets, _('Pkts.')) ];
+
+			if  (filterText) {
+				let filterTextExpressions = filterText.split(' ');
+				if (filterTextExpressions.some((element) => element.toUpperCase !== network && element.toUpperCase() !== protocol 
+						&& !(c.src.includes(element) || source.includes(element))
+						&& !(c.dst.includes(element) || destination.includes(element)))) {
```

Comment:
Can we skip the c.* checks? They're implied in `source` and `destination`, yes?

Reply:
Not exactly... 
- the `c.src` and `c.dst` contains the IP always
- the `source` and `destination` contains the IP or the HOSTNAME (if the DNS resolution button is enabled) and the PORT

So this is a simple way to filter by IP or hostname. If the DNS resolution is disabled it will filter by IP twice, but if I try to represent this in the condition in the filter to use it only when the DNS resolution is disabled it will be a more complicated function, so I decided to let it simple.